IDL/unions.idl

```diff
@@ -0,0 +1,444 @@
+#include "helpers/basic_inner_types.idl"

```

Comment:
I miss a union with `default` label.

Reply:
Added in [80f8b5b](https://github.com/eProsima/dds-types-test/pull/1/commits/80f8b5ba871562eb71479fbbe31a51e50321b1ca)